src/cascadia/inc/ControlProperties.h

```diff
@@ -16,10 +16,11 @@
     X(winrt::Microsoft::Terminal::Core::AdjustTextMode, AdjustIndistinguishableColors, winrt::Microsoft::Terminal::Core::AdjustTextMode::Never)
 
 // --------------------------- Control Appearance ---------------------------
-//  All of these settings are defined in IControlSettings.
+//  All of these settings are defined in IControlAppearance.
 #define CONTROL_APPEARANCE_SETTINGS(X)                                                                                                          \
     X(til::color, SelectionBackground, DEFAULT_FOREGROUND)                                                                                      \
     X(double, Opacity, 1.0)                                                                                                                     \
+    X(winrt::hstring, ColorSchemeName, L"Campbell")                                                                                             \

```

Comment:
I don't think we need this at all. `ICoreAppearance` should be able to handle all of the values for the color table. So in this layer, we shouldn't even have/need a concept of a "color scheme name" at all